src/content/1.7/development/internationalization/make-a-new-language-available-to-be-used-in-the-back-office.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Make a new language available to be used in the Back Office
+aliases:
+  - /1.7/development/make_a_new_language_available_to_be_used_in_the_back_office
+  - /1.7/development/make-a-new-language-available-to-be-used-in-the-back-office
+---
+
+# Make a new language available to be used in the Back Office
+
+Before adding a language in Prestashop, you must check if the language is available in [Crowdin](https://crowdin.com/project/prestashop-official).
+If the language is not available in [Crowdin](https://crowdin.com/project/prestashop-official), you have to contact [Louise Bonnard on Crowdin](https://crowdin.com/profile/LouiseBonnard) for opening a project in the concerned language. 
+After this prerequisite, adding a language in PrestaShop has two steps :
```

Comment:
_For that, it needs to be added as a new supported language in PrestaShop, and it takes two steps:_

Reply:
:ok_hand: